src/components/audio-player-controls/audio-player-controls.tsx

```diff
@@ -21,13 +21,42 @@ interface AudioPlayerControlsProps {
 const AudioPlayerControls = ({pluginConfig, player, onPluginsControlClick, showMorePluginsIcon}: AudioPlayerControlsProps) => {
   const playlist = useSelector((state: any) => state.engine.playlist);
   const ref = useRef<HTMLDivElement>();
+  const [selectedSpeedIndex, setSelectedSpeedIndex] = useState(1);
+  const speedValuesArray: Array<number> = [];
 
   useEffect(() => {
     ref.current?.setAttribute('tabindex', '0');
-  }, []);
+
+    const handleKeyDown = (event: KeyboardEvent) => {
+      if (!speedValuesArray.length) return;
+
+      if (event.shiftKey && event.key === '>') {
+        event.preventDefault();
+        setSelectedSpeedIndex(prev => {
+          const newIndex = Math.min(prev + 1, speedValuesArray.length - 1);
+          player.playbackRate = speedValuesArray[newIndex];
+          return newIndex;
+        });
+      } else if (event.shiftKey && event.key === '<') {
+        event.preventDefault();
+        setSelectedSpeedIndex(prev => {
+          const newIndex = Math.max(prev - 1, 0);
+          player.playbackRate = speedValuesArray[newIndex];
+          return newIndex;
+        });
+      }
+    };
+
+    window.addEventListener('keydown', handleKeyDown);
```

Comment:
that would effect all Kaltura players on page (in case if there more than one player), does this expected that on "<"/">" buttons click all audio players will change speed speed index?